rsts/deployment/deployment/multicluster.rst

```diff
@@ -132,21 +317,36 @@ file named ``secrets.yaml`` that looks like:
      name: cluster-credentials
      namespace: flyte
    type: Opaque
-   data:
-     cluster_1_token: {{ cluster 1 token here }}
-     cluster_1_cacert: {{ cluster 1 cacert here }}
-     cluster_2_token: {{ cluster 2 token here }}
-     cluster_2_cacert: {{ cluster 2 cacert here }}
-     cluster_3_token: {{ cluster 3 token here }}
-     cluster_3_cacert: {{ cluster 3 cacert here }}
-
-Create cluster credentials secret in the control plane cluster.
+   stringData:
```

Comment:
I'd suggest to use directly encoded data here since the new lines are error prone

Reply:
Right.
So you mean also modifying the previous command to avoid the `-D` and also using a `data` section, right?

Reply:
yes